okokyst_processing.py

```diff
@@ -267,16 +267,26 @@ def createHistoricalTimeseries(stationsList, CTDConfig):
 def createTimeseries(stationsList, CTDConfig):
     for station in stationsList:
         station.createTimeSection(CTDConfig)
-        station.createTimeseriesPlot(CTDConfig)
+        station.createTimeseriesPlot(CTDConfig,work_dir)
```

Comment:
Could we move `work_dir` into CTDConfig so that the user only needs to edit settings one place?